lib/rubocop/cop/flexport/engine_api_boundary.rb

```diff
@@ -145,6 +145,35 @@ module Flexport
       #     # (No direct associations to models in API-protected engines.)
       #   end
       #
+      # # "Engines" parameter
+      #
+      # The list of engine names parameter can be used instead of "EnginesPath" parameter
+      #
+      # The "Engines" parameter helps in the case when engines are placed in a project's root dir
+      #
+      # @example
+      #
+      #  in *.yml file:
+      #    Engines:
+      #      - my_engine1
+      #      - my_engine2
+      #
+      # # "EnginesPrefix" parameter
+      #
+      # Prefix/namespace of engine classes
+      #
+      # The "EnginesPrefix" parameter helps in the case when engine classes have namespace
+      #
+      # @example
+      #
+      #  module MyNamespace
+      #    class MyEngine::MyModel < ApplicationModel
+      #    end
+      #  end
+      #
+      #  in *.yml file:
+      #    EnginesPrefix: 'MyNamespace'
+      #
```

Comment:
Let's put this whole new section above the first `@example` above. We should also omit the `@example` blocks and instead use blocks like this:


        ```yml
        Engines
          - my_engine1
          -  my_engine2
        ```

The `@example` pragma indicates to the docs generator that the following code block is a Ruby snippet, as in the generated docs here:

https://docs.rubocop.org/rubocop/cops_naming.html